src/open_inwoner/openzaak/models.py

```diff
@@ -141,8 +141,20 @@ class ZaakTypeConfig(models.Model):
     )
 
     # actual config
+
+    # notifications
     notify_status_changes = models.BooleanField(default=False)
 
+    # documents
+    external_document_upload_url = models.URLField(
+        verbose_name=_("Document upload URL"),
+        blank=True,
+    )
+    document_upload_enabled = models.BooleanField(
```

Comment:
Does this have to do with external uploading?We have the same below for `ZaakTypeInformatieObjectTypeConfig` and its name is a bit confusing, at least for me.

Reply:
I think the Taiga ticket 1045 explains best how it is supposed to work (it was my ticket so you might not have seen this.).

Maybe the name is not ideal; we can rename to fit the actual implementation (easy to migrate).